fieldsolver/gridGlue.hpp

```diff
@@ -97,3 +111,69 @@ int getNumberOfCellsOnMaxRefLvl(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geomet
 void feedBoundaryIntoFsGrid(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
 			const std::vector<CellID>& cells,
 			FsGrid< fsgrids::technical, FS_STENCIL_WIDTH> & technicalGrid);
+
+/*Compute coupling DCCRG <=> FSGRID 
+
+  onDccrgMapRemoteProcess   maps fsgrid processes (key) => set of dccrg cellIDs owned by current rank that map to  the fsgrid cells owned by fsgrid process (val)
+
+  onFsgridMapRemoteProcess  maps dccrg processes  (key) => set of dccrg cellIDs owned by dccrg-process that map to current rank fsgrid cells 
+  onFsgridMapCells          maps remote dccrg CellIDs to local fsgrid cells
+*/
+
+// this function is declared here as it is a template function
+
+template <typename T, int stencil> void computeCoupling(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
+							const std::vector<CellID>& cells,
+							FsGrid< T, stencil>& momentsGrid,
+              std::map<int, std::set<CellID> >& onDccrgMapRemoteProcess,
```

Comment:
Could you just use the `onDccrgMapRemoteProcessGlobal` etc in this function without providing them as arguments?